data/mods/MMA/mutations.json

```diff
@@ -28,5 +28,11 @@
     "starting_trait": true,
     "valid": false,
     "cancels": [ "LIGHTWEIGHT" ]
+  },
+  {
+    "type": "muation",
```

Comment:
```suggestion
    "type": "mutation",
```

Reply:
Well shoot, I thought I fixed that. Guess not,  I will get a fix out ASAP